src/caffe/proto/caffe.proto

```diff
@@ -1277,6 +1364,11 @@ message V1LayerParameter {
   optional V0LayerParameter layer = 1;
 }
 
+
+message L1LossParameter {
+  optional bool debug_info = 1 [default = false];
+}
```

Comment:
Perfect - I was about to ask about this. @drendleman I'll let you do this since you're messing with the C++ code.